testdata/project-v3-addon/PROJECT

```diff
@@ -2,20 +2,4 @@ domain: testproject.org
 layout: go.kubebuilder.io/v3
 projectName: project-v3-addon
 repo: sigs.k8s.io/kubebuilder/testdata/project-v3-addon
-resources:
-- api:
-    crdVersion: v1
-  group: crew
-  kind: Captain
-  version: v1
-- api:
-    crdVersion: v1
-  group: crew
-  kind: FirstMate
-  version: v1
-- api:
-    crdVersion: v1
-  group: crew
-  kind: Admiral
-  version: v1
```

Comment:
Hi @varshaprasad96, 

Shows that have something broked with the addon partner. we might to need to do changes first in sigs.k8s.io/kubebuilder-declarative-pattern.

c/c @estroz

Reply:
Yes, the samples for v3-addon need to be regenerated after https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/pull/146 is merged